docs/instances/instance-to-instance.md

```diff
@@ -1,6 +1,6 @@
 # Communication between instances
 
-Instance-to-instance communication in the nio Platform uses standard [publish-subscribe patterns](/services/service-design-patterns/pub-sub.md) and is brokered by [Pubkeeper](/pubkeeper/README.md).
+Instance-to-instance communication in the nio Platform uses standard [publish-subscribe patterns](/services/service-design-patterns/pub-sub.html) and is brokered by [Pubkeeper](/pubkeeper/README.md).
```

Comment:
internal links should stay .md: https://toolchain.gitbook.com/faq.html#should-i-use-an-html-or-md-extensions-in-my-links
(paths and links get weird in gitbook, so best to follow recommendations!)